flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -1216,7 +1224,7 @@ public void testStateMigrationAfterChangingTTL() throws Exception {
         testKeyedValueStateUpgrade(initialAccessDescriptor, newAccessDescriptorAfterRestore);
     }
 
-    @Test
+    @TestTemplate
     public void testStateMigrationAfterChangingTTLFromEnablingToDisabling() {
```

Comment:
```suggestion
    void testStateMigrationAfterChangingTTLFromEnablingToDisabling() {
```
could be package private